src/providers/python.rs

```diff
@@ -220,7 +220,9 @@ impl PythonProvider {
         // Check for the engine database type in settings.py
         let re = Regex::new(r"django.db.backends.postgresql").unwrap();
 
-        app.find_match(&re, "/**/*.py")
+        let uses_pg =
+            app.find_match(&re, "/**/*.py")? || PythonProvider::uses_dep(app, "psycopg2")?;
```

Comment:
This should use a check dep function imo.

Reply:
It does. `PythonProvider::uses_dep`. The `find_match` is looking for a specific file name